src/main/java/xbony2/huesodewiki/infobox/parameters/LuminanceParameter.java

```diff
@@ -0,0 +1,30 @@
+package xbony2.huesodewiki.infobox.parameters;
+
+import net.minecraft.block.Block;
+import net.minecraft.item.ItemBlock;
+import net.minecraft.item.ItemStack;
+import xbony2.huesodewiki.api.infobox.IInfoboxParameter;
+
+public class LuminanceParameter implements IInfoboxParameter {
+
+	@Override
+	public boolean canAdd(ItemStack itemstack){
+		if(itemstack.getItem() instanceof ItemBlock){
+			Block block = ((ItemBlock) itemstack.getItem()).getBlock();
+			return block.getLightValue(block.getDefaultState(), null, null) > 0;
+		}
+		
+		return false;
+	}
+
+	@Override
+	public String getParameterName(){
+		return "luminance";
+	}
+
+	@Override
+	public String getParameterText(ItemStack itemstack){
+		Block block = ((ItemBlock) itemstack.getItem()).getBlock();
+		return String.valueOf(block.getLightValue(block.getDefaultState(), null, null));
```

Comment:
This is an int, right? I prefer Integer.toString since I've been using that elsewhere.

Reply:
Switched it to `Integer.toString`.